packages/console-types/src/types/catalog/well-known-items/proxy/index.ts

```diff
@@ -47,7 +47,7 @@ const resourcesSchema = {
             additionalProperties: false,
             properties: {
               description: descriptionSchema,
-              host: { pattern: '^[a-z]([-a-z0-9]*[a-z0-9])?(\\:\\d{2,4})$', type: 'string' },
+              host: { pattern: '^[a-z]([-a-z0-9]*[a-z0-9])?(:\\d{2,4})$', type: 'string' },
```

Comment:
shouldn't this regex match the one used by services?

https://github.com/mia-platform/console-sdk/blob/main/packages/console-types/src/types/services.ts#L781

otherwise one may define a proxy in the catalog that can't be created in the Console

Reply:
Updated it to use the service one